contribs/gmf/src/directives/objecteditingtools.js

```diff
@@ -191,11 +192,16 @@ gmf.ObjecteditingtoolsController = function($scope, ngeoDecorateInteraction,
    */
   this.triangleAngle = Math.PI / 180 * 90; // 90 degrees
 
+  var oeToolsOptions = /** @type {gmfx.ObjectEditingToolsOptions} */ (
+    $injector.has('gmfObjectEditingToolsOptions') ?
+      $injector.get('gmfObjectEditingToolsOptions') : {});
+
   /**
    * @type {number}
    * @export
    */
-  this.triangleRadius = 100;
+  this.triangleRadius = oeToolsOptions.regularPolygonRadius !== undefined ?
+    oeToolsOptions.regularPolygonRadius : 100;
```

Comment:
Just a little point:  Can you add the default value like this:

https://github.com/camptocamp/ngeo/blob/699a903ba78334f01d3255458b74477764cc9576/contribs/gmf/src/services/thememanager.js#L12

I think that's more compact and we don't need to load `$injector`. (Don't do it if you see any problem about this.)